exercises/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,15 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+function changeToUppercase(name) {
+  return (upperCasename = name.toUpperCase());
```

Comment:
(optional) it would be clearer if you created a let here and then called it on return.

Reply:
thank you @luan-murilo-de-oliveira-e-souza , I agree with you.